<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DeleteData</class>
 <widget class="QWidget" name="DeleteData">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>478</width>
    <height>982</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="RangeSections">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>291</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>Удалить диапазон секций</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>С</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>20</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>По</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxFromRemovableSection">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxToRemovableSection">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>40</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonDeleteRangeSections">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>70</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Удалить </string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="Section">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>10</y>
     <width>161</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>Удалить секцию</string>
   </property>
   <widget class="QComboBox" name="comboBoxRemovableSection">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Имя секции</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonDeleteOneSection">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>70</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Удалить </string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="Key">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>110</y>
     <width>161</width>
     <height>160</height>
    </rect>
   </property>
   <property name="title">
    <string>Удалить ключ</string>
   </property>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Имя секции</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxNameSectionDeleteKey">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>70</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Имя ключа</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonDeleteKey">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>130</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Удалить </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditRemovableKey">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>120</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="ValueInKey">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>291</width>
     <height>160</height>
    </rect>
   </property>
   <property name="title">
    <string>Удалить одно из значений ключа</string>
   </property>
   <widget class="QComboBox" name="comboBoxNameSectionForDeleteValue">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Имя секции</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Имя ключа</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>70</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Значение</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonDeleteValueInKey">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>130</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Удалить </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditNameKeyForDeleteValue">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditRemovableValue">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>90</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="Comment">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>161</width>
     <height>100</height>
    </rect>
   </property>
   <property name="title">
    <string>Удалить комментарий</string>
   </property>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Имя секции</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxNameSectionForDeleteComment">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonDeleteComment">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>70</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Удалить </string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>380</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Контент секции после удаления</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>400</y>
     <width>460</width>
     <height>570</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonDeleteData">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>280</y>
     <width>160</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Удалить все данные
 файла</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonDeleteFile">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>330</y>
     <width>160</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Удалить  файл</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
